pcsx2/gui/ConsoleLogger.h

```diff
@@ -220,3 +223,29 @@ class ConsoleLogFrame : public wxFrame
 
 	void OnLoggingChanged();
 };
+
+template<typename ... Args>
+void OSDlog(ConsoleColors color, bool console, const std::string& format, Args ... args) {
+	if(!GSosdLog) return;
```

Comment:
The message should still be printed to the console even if the plugin doesn't support OSD logging.